### ISSUE TITLE
Name of custom MaterialPass for ColorStripeMaterialVariables as string parameter?

### ISSUE THREAD

**Issue by federgraph** [helix-toolkit/helix-toolkit#1576]
The Wpf.SharpDX _CustomShaderDemo_ shows how to use a custom constant buffer with a custom pixel shader and PointLineMaterial. 

This is close to what I need: A **custom constant buffer for my custom vertex shader**, to be used with **ColorStripeMaterial**.

But I needed to change _ColorStripeMaterialVariables_ class and pass the name of the custom ShaderPass into the constructor, in a similar fashion.

Should this change be made part of the library?

See #1358 and #1360 and my [example](https://github.com/federgraph/helix-toolkit/commit/c8ccfbc0e61c34297a40c86de900e40d7cb96709).

**Comment by federgraph** [comment]
Update: I managed to compile my test project using current NuGet package - HelixToolkit.Wfp.SharpDX.

I ( just ) needed to clone more code from ColorStripeMaterial to my project as MyColorStripeMaterial, and apply the small change there.
- MyColorStripeMaterialVariables : MaterialVariable
- MyColorStripeMaterialCore : MaterialCore
- MyColorStripeMaterial : Material

Basically, I have created a new material, instead of using ColorStripeMaterial. If this will be easier in the futue I will test it out!
